NOTICE

```diff
@@ -3,3 +3,12 @@ Copyright 2024 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+Apache Iceberg
+Copyright 2017-2022 The Apache Software Foundation
+
+Apache Paimon
+Copyright 2023-2024 The Apache Software Foundation
+
+Apache Flink
+Copyright 2014-2023 The Apache Software Foundation
```

Comment:
```suggestion
Copyright 2014-2024 The Apache Software Foundation
```

Reply:
I copied this notice content from https://github.com/apache/iceberg/blob/main/NOTICE.

I am not sure if we should change it.

Reply:
It's OK to set the correct copyright year. And we can also submit pull requests to correlative projects to fix this issue.

Reply:
Thanks for the information, I have changed it.